pkg/controller/dataimportcron-controller.go

```diff
@@ -1077,6 +1077,17 @@ func addDataImportCronControllerWatches(mgr manager.Manager, c controller.Contro
 		return err
 	}
 
+	if err := c.Watch(source.Kind(mgr.GetCache(), &corev1.PersistentVolumeClaim{}),
```

Comment:
May want to place this above the snapshot watch, since it has a https://github.com/kubevirt/containerized-data-importer/blob/4e7e6833fa7bbf9566c4abd2996c3a231b2a034f/pkg/controller/dataimportcron-controller.go#L1062-L1063

Reply:
Good catch, fixed.